app/src/test/java/io/apicurio/registry/SerdeMixTest.java

```diff
@@ -57,9 +57,10 @@ public void testVersions(Supplier<RegistryService> supplier) throws Exception {
         Schema schema = new Schema.Parser().parse("{\"type\":\"record\",\"name\":\"myrecord5\",\"fields\":[{\"name\":\"bar\",\"type\":\"string\"}]}");
         int id = confClient.register(subject, schema);
         confClient.reset();
-        
+
         this.waitForArtifact(subject);
 
+        this.waitForGlobalId(id);
```

Comment:
What does waiting for the global id accomplish that waiting for the artifact doesn't?

Reply:
for some reason the `confClient.getById(id);` fails if you don't wait

Reply:
@carlesarnal Can you verify this locally?  Also if you could try it with line 61 removed as well.  I don't see why we would need both of these waits.

Reply:
Hi @famartinrh , @EricWittmann, makes sense that we need to add here the `waitForGlobalId` since the id argument of the `getById` operation is a global id. I'm not sure about the `waitForArtifact` but, I guess that we need it since we are trying to perform an `updateArtifact` operation that otherwise might fail.

Reply:
Thanks @carlesarnal